core/src/test/java/kafka/admin/UserScramCredentialsCommandTest.java

```diff
@@ -0,0 +1,213 @@
+import kafka.test.annotation.Type;
+import kafka.test.junit.ClusterTestExtensions;
+import kafka.utils.Exit;
+import org.apache.kafka.test.NoRetryException;
+import org.apache.kafka.test.TestUtils;
+import org.junit.jupiter.api.extension.ExtendWith;
+import scala.Console;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.OptionalInt;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SuppressWarnings("dontUseSystemExit")
+@ExtendWith(value = ClusterTestExtensions.class)
+@ClusterTestDefaults
+public class UserScramCredentialsCommandTest extends BaseRequestTest {
+    private static final String USER1 = "user1";
+    private static final String USER2 = "user2";
+
+    @Override
+    public int brokerCount() {
+        return 1;
+    }
+
+    static class ConfigCommandResult {
+        public final String stdout;
+        public final OptionalInt exitStatus;
+
+        public ConfigCommandResult(String stdout) {
+            this(stdout, OptionalInt.empty());
+        }
+
+        public ConfigCommandResult(String stdout, OptionalInt exitStatus) {
+            this.stdout = stdout;
+            this.exitStatus = exitStatus;
+        }
+    }
+
+    private ConfigCommandResult runConfigCommandViaBroker(String...args) {
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        String utf8 = StandardCharsets.UTF_8.name();
+        PrintStream printStream;
+        try {
+            printStream = new PrintStream(byteArrayOutputStream, true, utf8);
+        } catch (UnsupportedEncodingException e) {
+            throw new RuntimeException(e);
+        }
+        AtomicReference<OptionalInt> exitStatus = new AtomicReference<>(OptionalInt.empty());
+        Exit.setExitProcedure((status, __) -> {
+            exitStatus.set(OptionalInt.of((Integer) status));
+            throw new RuntimeException();
+        });
+
+        List<String> commandArgs = new ArrayList<>(Arrays.asList("--bootstrap-server", bootstrapServers(listenerName())));
+        commandArgs.addAll(Arrays.asList(args));
+        try {
+            Console.withOut(printStream, () -> {
+                ConfigCommand.main(commandArgs.toArray(new String[0]));
+                return null;
+            });
+            return new ConfigCommandResult(byteArrayOutputStream.toString(utf8));
+        } catch (Exception e) {
+            return new ConfigCommandResult("", exitStatus.get());
+        } finally {
+            printStream.close();
+            Exit.resetExitProcedure();
+        }
+    }
+
+    @ClusterTests({
```

Comment:
As we define all types in the class-level, `ClusterTests` can be replaced by `ClusterTest`